css/counter.css

```diff
@@ -0,0 +1,59 @@
+:root {
+    --light-blue: rgb(223, 235, 250);
+    --medium-blue: rgb(139, 186, 247);
+    --dark-blue: rgb(16, 74, 175)
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+main {
```

Comment:
it looks rather like `main.css` stuff than `counter.css` 😄 I mean styling `main` element is global thing not exactly related to the counter itself - positioning some component inside the view is a job for the view and not for component :)